firefox-ios/Storage/Rust/RustLogins.swift

```diff
@@ -647,6 +648,33 @@ public class RustLogins: LoginsProtocol {
         }
     }
 
+    func searchLoginsWithQuery(_ query: String?, completionHandler: @escaping (Result<EncryptedLogin?, Error>) -> Void) {
```

Comment:
could you please add `public` to the start of this method so it's consistent with the rest of this class?
Or you could remove the other instances of public on the methods in this class and just leave it at the class declaration level, either way, the styling should be consistent.

Reply:
Hi @dataports, thanks for your feedback! The method has been made public as per your suggestion to maintain consistency within the class 🙏